opencensus/trace/ext/django/middleware.py

```diff
@@ -155,10 +160,23 @@ def __init__(self, get_response=None):
                 endpoint=_endpoint,
                 transport=transport)
         elif self._exporter.__name__ == 'JaegerExporter':
-            _service_name = self._get_service_name(settings.params)
+            _service_name = settings.params.get(
+                JAEGER_EXPORTER_SERVICE_NAME,
+                self._get_service_name(settings.params))
+            _jaeger_host_name = settings.params.get(
+                JAEGER_EXPORTER_HOST_NAME, None)
+            _jaeger_port = settings.params.get(
+                JAEGER_EXPORTER_PORT, None)
+            _jaeger_agent_host_name = settings.params.get(
+                JAEGER_EXPORTER_AGENT_HOST_NAME, 'localhost')
+            _jaeger_agent_port = settings.params.get(
+                JAEGER_EXPORTER_AGENT_PORT, 6831)
             self.exporter = self._exporter(
                 service_name=_service_name,
-                transport=transport)
+                host_name=_jaeger_host_name,
+                port=_jaeger_port,
+                agent_host_name=_jaeger_agent_host_name,
```

Comment:
Do we want agent_host_name and agent_port in Flask Middleware?

Reply:
Yep, can add to this PR if you like?